llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1053,6 +1037,10 @@ FastMathFlags SPIRVToLLVM::getFastMathFlags(SPIRVValue *bv) {
 
     fmf.setNoSignedZeros();
   }
+
+  if (m_shaderOptions->disableFastMathFlags)
+    fmf &= buildFastMathFlags(~0U ^ m_shaderOptions->disableFastMathFlags);
```

Comment:
Is this the same as `~m_shaderOptions->disableFastMathFlags`?

Reply:
Yes, I've changed it.